source/PreferencesPanel.h

```diff
@@ -69,7 +75,10 @@ class PreferencesPanel : public Panel {
 
 	Point hoverPoint;
 	int hoverCount = 0;
+	std::string selectedItem;
 	std::string hoverItem;
+	std::string oldSelectedItem;
+	std::string oldHoverItem;
```

Comment:
Well now these are unused too.

Reply:
I feel like C++ should give warnings for unused variables :P